pinot-query-runtime/src/test/resources/queries/FromExpressions.json

```diff
@@ -57,9 +57,14 @@
       },
       {
         "psql": "7.2.1.1",
-        "ignored": true,
+        "comments": "select all doesn't work because h2 output the rows in different order from Pinot and test framework doesn't consider the column order for rows and blindly assume they are in the same order",
```

Comment:
can we add a test for this with the outputs specified?

Reply:
Unfortunately no because ts columns is also selected, that value cannot be specified.